src/config.ts

```diff
@@ -43,6 +43,9 @@ export function defaultConfig(
     // Replace the contents of a translation file (if it exists) with the generated one (default value is false, in which case files are merged)
     replace: false,
 
+    // Delete missing keys from existing translation files
+    deleteMissingKeys: false,
```

Comment:
Please see [this comment](https://github.com/ngneat/transloco-keys-manager/pull/89#discussion_r1050829480)